guides/creating-a-custom-form-field-control.md

```diff
@@ -0,0 +1,337 @@
+It is possible to create custom form field controls that can be used inside `<mat-form-field>`. This
+can be useful if you need to create a component that shares a lot of common behavior with a form
+field, but adds some additional logic.
+
+In order to learn how to build custom form field controls, let's start with a simple input component
+that we want to work inside the form field. For example, a phone number input that segments the
+parts of the number into their own inputs. (Note: this is not intended to be a robust component,
+just a starting point for us to learn.)
+
+<!--
+  TODO(mmalerba): Link to Plunker/StackBlitz with complete example code after beta 12.
+  (Beta 11 is missing some of the features used in the example).
+-->
```

Comment:
We can embed live examples in guides too, so it doesn't have to be an external link.